logger/scenario_logger_msgs/CMakeLists.txt

```
@@ -1,30 +1,23 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(scenario_logger_msgs)
 
-find_package(catkin REQUIRED COMPONENTS
-    message_generation
-    std_msgs
-)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
-add_message_files(
-  DIRECTORY msg
-  FILES
-  Log.msg
-  Level.msg
-  MetaData.msg
-  LoggedData.msg
-)
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
```

I think you're missing the the `-Wno-unused-parameter`. As discussed here: https://github.com/tier4/AutowareArchitectureProposal/issues/77. But I realise this is probably not necessary given that only messages are being built.
```suggestion
  add_compile_options(-Wall -Wextra -Wpedantic -Wno-unused-parameter)
```